BitwardenShared/UI/Platform/Application/Appearance/UI.swift

```diff
@@ -99,10 +99,10 @@ public enum UI {
         )
         UISearchBar.appearance().tintColor = Asset.Colors.textInteraction.color
         // Explicitly tint the image so that it does not assume the tint color assigned to the entire search bar.
-        let image = Asset.Images.cancelRound.image
+        let image = Asset.Images.circleX24.image
         let tintedImage = image.withTintColor(Asset.Colors.textSecondary.color, renderingMode: .alwaysOriginal)
         UISearchBar.appearance().setImage(tintedImage, for: .clear, state: .normal)
-        UISearchBar.appearance().setImage(Asset.Images.magnifyingGlass.image, for: .search, state: .normal)
+        UISearchBar.appearance().setImage(Asset.Images.search24.image, for: .search, state: .normal)
```

Comment:
❓ Should these be 16x16 icons? They look a little big.
<img width="861" alt="Screenshot 2024-10-21 at 1 25 05 PM" src="https://github.com/user-attachments/assets/0e26f769-7471-4202-aee3-06a9133a228c">

Reply:
yeah, mistake on my side.  got em fixed up